flaskbb/utils/populate.py

```diff
@@ -130,8 +130,10 @@ def update_settings_from_fixture(fixture, overwrite_group=False,
                     )
 
                 setting.save()
-                updated_settings[group] = []
-                updated_settings[group].append(setting)
+                try:
```

Comment:
Using `dict.setdefault` or a `collections.defaultdict(list)` would be preferable here.

Reply:
Ah… learning Python in 2007 and then barely touching it for almost ten years really bit me in the backside here. Back then, I read something along the lines of collections.defaultdict not being backwards-compatible and
```
try:
    Something
except SpecificError:
    askForgiveness
```
being more Pythonic anyway. Probably was a typical case of: If I _can’t_ have it then I also don’t _want to_ because (insert dubious reason).

You’re probably going to have to give me a few more of those until I fully realize that we live in 2017 now and will drop Python 2.7 support soon.

Anyway, should be fixed now.